### PR TITLE
[Binding addons starting with "E"] Correct linter errors only

### PR DESCRIPTION
**Pull request by bgilmer77**
Signed-off-by: Brad Gilmer <brad@gilmer.tv>

### CODE REVIEW

File: addons/binding/org.openhab.binding.exec/README.md [hunk 1]
```diff
@@ -10,42 +10,45 @@ Currently, the binding supports a single type of Thing, being the ```command```
...
 All Things support the following channels:
 
-| Channel Type ID | Item Type    | Description  |
-|-----------------|------------------------|--------------|----------------- |------------- |
-| input | String       | Input parameter to provide to the command |
-| output | String       | Output of the last execution of the command |
-| exit | Number       | The exit value of the last execution of the command |
-| run | Switch       | Send ON to execute the command and the current state tells whether it is running or not |
-| lastexecution | DateTime       | Time/Date the command was last executed, in yyyy-MM-dd'T'HH:mm:ss.SSSZ format |
+| Channel Type ID | Item Type | Description                                                                             |   |   |
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.exec/README.md]: Could you remove the last two columns?

Comment (bgilmer77) [reply addons/binding/org.openhab.binding.exec/README.md]: Agree - This will be in my next commit.

**Comment by bgilmer77** [comment]
Comment addressed - ready to merge?

**Comment by martinvw** [comment]
Thanks for your work on the docs!

**Pull request merged**